src/persist-types/src/codec_impls.rs

```diff
@@ -214,14 +217,14 @@ impl Schema<Vec<u8>> for VecU8Schema {
     type Decoder<'a> = SimpleDecoder<'a, Vec<u8>, Vec<u8>>;
 
     fn columns(&self) -> Vec<(String, DataType, StatsFn)> {
```

Comment:
mentioned to ben offline, we could potentially replace this Vec with DynStructCfg to make the connection more clear. worth it? (i lean toward yes)

Reply:
Also lean towards yes, though I think the usage is _reasonably_ clear from context, so it is slight.